docker-compose.yml

```diff
@@ -141,20 +141,24 @@ services:
       - ./test:/app/test
       - ./features:/app/features
 
-  react-native-cli:
-    build:
-      context: .
-      dockerfile: dockerfiles/Dockerfile.react-native-cli
+  react-native-cli-tool-maze-runner:
+    # TODO Use Maze Runner release
+    image: 855461928731.dkr.ecr.us-west-1.amazonaws.com/maze-runner:integration-interactive_cli-cli
     environment:
       DEBUG:
+      VERBOSE:
       REACT_NATIVE_VERSION:
       NETWORK_NAME: "${BUILDKITE_JOB_ID:-react-native-cli-maze-runner}"
+      REGISTRY_URL:
+      REG_NPM_EMAIL:
+      REG_BASIC_CREDENTIAL:
     networks:
       default:
         aliases:
           - maze-runner
     volumes:
       - /var/run/docker.sock:/var/run/docker.sock
+      - ./test/react-native-cli/features/:/app/features/
```

Comment:
Doesn't this mean any changes that the tests make will be reflected in your local file system? The tests will make _a lot_ of changes to files, so we probably don't want this unless there's a way to exclude the `fixtures` directory

```suggestion
```

Reply:
The intention is that the Maze Runner container (which this configures) only reads the `.feature` files.  The test fixture directories are copied into their own Docker images as part of running the tests.  So I wouldn't expect this to result in changes to the local file system when running locally - but will check!

Reply:
Yeah, I think you're right 👍